identity/app/views/completeConsents.scala.html

```diff
@@ -43,10 +44,10 @@ <h1 class="identity-title--small">Before you go...</h1>
                         The Guardian's newsletters include content from our website, which may be funded by outside parties. Newsletters may also display information about Guardian News and Media's other products, services or events (such as Guardian Jobs or Masterclasses), chosen charities or online advertisements.
                     </p>
                 </div>
-            </form>
+            </div>
         }
         <aside class="identity-forms-message__body">
-            <a class="manage-account__button manage-account__button--secondary" href="@LinkTo(returnUrl)">
+            <a class="manage-account__button manage-account__button--secondary" data-link-name="complete-consents : cta-bottom : @LinkTo(returnUrl)" href="@LinkTo(returnUrl)">
```

Comment:
this does seem a little overzealous - is it not enough to just check the place people end up after actually clicking the link?

Reply:
I'm happy to remove this after a week or so but at this point I suspect that place might be hitting a redirect (not shown in DL) which would make this very useful data for bugfixing purposes

Reply:
Ok good call, let’s do that